src/compiler/checker.ts

```diff
@@ -9520,27 +9541,36 @@ namespace ts {
             return result;
         }
 
-        function getExpandedParameters(sig: Signature): readonly Symbol[] {
+        function getExpandedParameters(sig: Signature, skipUnionExpanding?: boolean): readonly (readonly Symbol[])[] {

```

Comment:
I understand this change on its surface, but I don’t immediately understand why it was necessary. Can you point me to the test case(s) where this matters?

Reply:
It returns of array of arrays so it can return, to the language service, a list of "psuedooverloads" - one for each rest union tuple member. The flag exists to disable that behavior for node serialization, where we'd like to keep it as the single overload.

Reply:
Got it, so this only comes into play in signature help (and possibly quick info or whatever)?

Reply:
Yep.